lib/screens/v2/candidate_generator/dup/alerts.ex

```diff
@@ -254,21 +254,21 @@ defmodule Screens.V2.CandidateGenerator.Dup.Alerts do
     end
   end
 
-  # Given an alert, see if any of its informed entities match a list of stops-of-interest (called stop_matchers here).
-  # If it has an informed entity on the list, return its branch.
-  @spec get_branch_if_entity_matches_stop(
+  # Given an alert, see if its informed entities match a list of stops-of-interest (called stop_matchers here).
+  # If a stop matcher is found, the branch is added the the returned list.
+  @spec get_branches_if_entity_matches_stop(
           Alert.t(),
           list(%{branch: String.t(), stop: String.t()})
         ) ::
           atom()
```

Comment:
Need to update this now that you return a list. 

```suggestion
          [atom()]
```

Also, should this be `String.t()` instead of `atom()`? `branch` is a string in the `stop_matchers` you give to the function above.

Reply:
ah yes! I believe this is why Dialyzer was complaining. 👍